### PR TITLE
[AzureRT] Update

### PR DESCRIPTION
**Pull request by hyonholee**
ARM Compute:
- Add Set-AzureRmBginfoExtension cmdlet
- Make WinRMCertificateUrl parameter mandatory when Set-AzureRmVMOperatingSystem cmdlet is performed with WinRMHttps switch

ASM Compute:
- Fix the issue when adding a new VM without a data disk
- Add ExtensionId parameter for all extension cmdlets.
- Expose RemoteAccessCertificateThumbprint property for Get-AzureVM cmdlet

This PR includes Network update with Swagger-generated library.

### CODE REVIEW

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/379/

File: src/ResourceManager/Compute/Commands.Compute.Test/packages.config [hunk 1]
```diff
@@ -19,7 +18,9 @@
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.18.206251556" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
   <package id="Microsoft.Rest.ClientRuntime" version="1.4.1" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="0.11.0" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="0.9.3" targetFramework="net45" />
```
Comment (jianghaolu) [inline src/ResourceManager/Compute/Commands.Compute.Test/packages.config]: @hyonholee Is there any reason this needs to be downgraded from 0.11.0 to 0.9.3?

Comment (jianghaolu) [inline src/ResourceManager/Compute/Commands.Compute.Test/packages.config]: Also in `Commands.Network.Test`

Comment (hyonholee) [inline src/ResourceManager/Compute/Commands.Compute.Test/packages.config]: Yes.  When I ran the tests, I got a System.IO.FileLoadException during loading this dll.  After I downgraded to 0.9.3, it worked.

**Comment by jianghaolu** [comment]
on-demand passing: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/393/. Merging

**Pull request merged**